pkg/scheduler/framework/plugins/noderesources/fit.go

```diff
@@ -303,12 +303,16 @@ func (f *Fit) isSchedulableAfterPodChange(logger klog.Logger, pod *v1.Pod, oldOb
 // isResourceScaleDown checks whether the resource request of the modified pod is less than the original pod
 // for the resources requested by the pod we are trying to schedule.
 func (f *Fit) isResourceScaleDown(targetPod, originalOtherPod, modifiedOtherPod *v1.Pod) bool {
```

Comment:
Can you change the parameter names from originalOtherPod to originalPod? (Since now maybe targetPod == originalPod) ditto for modifiedOtherPod.